src/systems.rs

```diff
@@ -63,7 +63,7 @@ pub struct TouchId(pub Option<u64>);
 pub struct InputResources<'w, 's> {
     #[cfg(all(feature = "manage_clipboard", not(target_os = "android")))]
     pub egui_clipboard: Res<'w, crate::EguiClipboard>,
-    pub keyboard_input: Res<'w, Input<KeyCode>>,
+    pub keyboard_input: Res<'w, ButtonInput<KeyCode>>,
```

Comment:
We'll want to read the event `KeyboardInput`, as `KeyCode` in bevy is no longer layout dependant.

Maybe it could be done in a followup issue?

Reply:
@Vrixyz do we want these inputs to be layout dependent? We are reading them only to check key modifiers (such as shift/ctrl/etc).

Reply:
Modifiers aren't exempt from keyboard layout, prominently many layouts have AltGraph instead of AltRight. Less industry-standard but still out there [Neo](https://upload.wikimedia.org/wikipedia/commons/2/2a/Kommerziell_produzierte_NEO-Tastatur_2017.JPG) replaces capslock and the key right of left shift (that US layouts don't have) with modifiers.

It may not have a huge practical impact and people with those kinds of layout are already accustomed to additional pain (compared to say plain dvorak) but consistently using logical mappings sounds like a good idea.

Reply:
@eerii you might be interested in parts of that diff: https://github.com/Vrixyz/bevy_egui/pull/4/commits/86650bb70d55bdc2853e0e9469da5bde3f75c2d5 ; I think that's not the whole diff needed to use logical key everywhere, but it was enough to get me unlock on my particular issue.

I look forward to finally consolidate bevy_egui to non-qwerty layouts :D 

(Just when I ordered a qwerty keyboard 🥲)

Reply:
I updated the branch to match main and to track the latest bevy main, and added your diff. It seems to work and while I'm not sure where else to look to migrate logical input it seems an improvement! Thanks ^^

Reply:
It's a bit of a shame that Bevy 0.13 doesn't have `ButtonInput<Key>` for logical keys

Reply:
I've just re-implemented the modifiers logic with logical key codes, I'll keep this for review till the evening and then merge it. I also hope to tackle the other input-related issues shortly after and then release a new version.